src/visual.ts

```diff
@@ -227,8 +267,10 @@ export class BulletChart implements IVisual {
     private static value1dot4: number = 1.4;
```

Comment:
These variable names looks really strange. Can you refactor this, giving the meaningful naming, e.g. `buletChartBarSpacing`. This name should represent something. Maybe you need to gather it in to some structure if all of these numbers represent some single idea (creating an enum, or something else if required).

Reply:
can you do the same for other variables here? `value2, value12....` all of them have no sense.

Reply:
Done.